### PR TITLE
Add snippet for Collection Group Query

### PR DESCRIPTION
**Pull request by BenWhitehead**
* Bump `@google-cloud/firestore` to `^1.3.0` which includes the new
  functionality

### CODE REVIEW

File: firestore/main/index.js [hunk 1]
```diff
@@ -1121,4 +1180,8 @@ describe('Firestore Smoketests', () => {
   it('should delete the whole collection', () => {
     return deleteCollection(db, 'cities', 50);
   });
+
+  it('should find all museums when querying a collection group', () => {
+    return collectionGroupQuery();
```
Comment (samtstern) [inline firestore/main/index.js]: Make sure to pass in the db here

Comment (BenWhitehead) [reply firestore/main/index.js]: Good catch, will update.

**Comment by samtstern** [comment]
@BenWhitehead so version `1.3.0` is public and has CG queries in it already?  Do they work against prod?

**Comment by BenWhitehead** [comment]
Version [`1.3.0`](https://github.com/googleapis/nodejs-firestore/releases/tag/v1.3.0) was released last friday. I'm told prod is ready for the clients, though haven't personally verified.

**Comment by samtstern** [comment]
Closed in favor of #43

**Pull request merged**